### DISCUSSION TITLE
DBG_PRINT and print_level

### DISCUSSION THREAD

**Discussion by danieloliveira56** [coin-or/Ipopt#567] [Q&A] [answered]
How can I make the following line print?

[`DBG_PRINT((1, "s_min = %g, curr_mu = %g, min_slack=%g\n", s_min, ip_data_->curr_mu(), min_slack));`
](https://github.com/coin-or/Ipopt/blob/a62679dd2cd292a9e3aa64ad05df37f5adfdece5/src/Algorithm/IpIpoptCalculatedQuantities.cpp#L478)

I tried print_level 8 and it didn't work. If I try a higher value the log gets too large for my problem.

**Comment by svigerske** [comment, marked as answer]
These are only enabled when Ipopt has been configured with `--with-ipopt-checklevel=1` (or higher; https://coin-or.github.io/Ipopt/INSTALL.html#CONFIGURE_FLAGS).
